common/src/main/resources/messages/messages.properties

```diff
@@ -186,7 +186,7 @@ cluster.pgw.unhealthy.sync.started=Cloudera Manager is unreachable, cluster sync
 cluster.gateway.change=Starting primary gateway change
 cluster.gateway.changed.successfully=Primary gateway successfully changed to {0}
 cluster.gateway.change.failed=Primary gateway change failed. Reason: {0}
-cluster.ambari.cluster.decommissioning.time=The decommission of the Data nodes will take approximately {0}.
+cluster.ambari.cluster.decommissioning.time=The decomission of the Data nodes will take approximately {0}.
```

Comment:
revert this please.

Reply:
Will do sorry I didn't think about how this could cause a negative impact.